src/derivatives/05-diffusion-derivatives.md

```diff
@@ -31,340 +31,810 @@ Template:
+                "ARDFudgeFactor": 1.0,
+                "Fibers": 3
+            },
+            "BootstrapParameters": {
+                "Burnin": 1000,
+                "Jumps": 1250,
+                "SampleEvery": 25
+            }
+        },
+        "OrientationEncoding": {
+            "Type": "scalar"
+        }
+    }
 
-If `AntipodalSymmetry` is True, then no constraints are imposed with respect
-to the domain on the 2-sphere in which orientations may be specified;
-for instance, 3-vectors { 0.57735, 0.57735, 0.57735 } and
-{ -0.57735, -0.57735, -0.57735 } are both permissible and equivalent to one
-another.
+    Notes:
+
+    -   Here the descriptors "`merged`" and "`mean`" have been utilised
+        to distinguish between the comprehensive set of all bootstrap realisations
+        and the computed mean statistics of parameters across all realisations respectively,
+        as this is the terminology utilised by the FSL software itself.
+        These labels are not however a part of the specification.
+
+    -   Care must be taken for images of greater than three dimensions
+        where additional dimensions do *not* encode anisotropy information:
+
+        -   In image `"*_desc-mean*_param-vf_*"`,
+            the fourth image axis encodes scalar information across stick components.
+            Since this is *not* coefficients in some orientation encoding,
+            but the image possesses more than three axes,
+            field `"OrientationEncoding"["Type"]` MUST be specified as `"scalar"`.
+
+        -   Image `"*_param-vfsum_*"`is a three-dimensional image,
+            and therefore the fact that it encodes a scalar parameter
+            can be robustly inferred without reference to metadata information.
+
+        -   In image `"sub-01_model-bs_desc-merged_param-vf_dwimap.json"`,
+            there are two extra image dimensions beyond the three spatial dimensions:
+            the fourth image axis encodes across the multiple stick components per voxel,
+            and the fifth axis encodes realisations across bootstraps.
+            it is therefore necessary to explicitly specify
+            that the parameter being encoded in the data,
+            being the volume fractions of individual stick components,
+            do not have any anisotropy,
+            and therefore field `"OrientationEncoding"["Type"]` MUST be specified as `"scalar"`.
+
+    -   TODO CONFIRM WHETHER WE WANT TO ENFORCE THIS OR INCLUDE EXPLICIT METADATA FIELDS
+        THAT INDICATE THE AXIS / AXES OF DIRECTION ENCODING VS. BOOTSTRAP REALISATIONS
+
+    -   TODO DISCUSS
+        The example presented here is not necessarily a unique solution
+        for the storage of the outcomes of the FSL `bedpostx` command as BIDS Derivatives:
+
+        -   WOULD SPLITTING STICK COMPONENTS ACROSS NIFTIS
+            REQUIRE A NEW ENTITY BY WHICH TO INDEX THEM?
+            OR JUST GIVE THEM EG. `_param-spherical1`, `_param-spherical2`?
```

Comment:
I haven't used bedpostx in a really long while, and even then only very superficially. I would love to get input from users of the tool who can tell us more about how they anticipate using these files and what they'd prefer.

Reply:
Perhaps defer dedicated discussions to #93 / #94.